lib/main.js

```diff
@@ -29,6 +29,10 @@ module.exports = {
     focusEditorAfterCompile: {
       type: 'boolean',
       "default": false
+    },
+    compileCjsx: {
+      type: 'boolean',
+      "default": false
```

Comment:
Oh wow. I must have add this compiled js version into source last time. This isn't necessary but I'll remove this file in separate commit.